tasks/managed/embargo-check/embargo-check.yaml

```diff
@@ -99,14 +99,31 @@ spec:
             fi
             set -x
 
-            # Perform additional checks only for Vulnerability issues for issues.redhat.com
+            # Perform additional checks only for issues for issues.redhat.com
```

Comment:
This reads wrong. Perhaps "for issues on issues.redhat.com"?

Reply:
fixed